### PR TITLE
Gem collections

### PR DESCRIPTION
**Pull request by jgersak**

### CODE REVIEW

**Review by zigaLuksic — APPROVED** [review]

File: GEM-data/gem-collections.ipynb [hunk 1]
```diff
@@ -51,6 +51,36 @@
...
   },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "e390008c-a274-46da-8521-b8f1e8c9e78a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def templat_evalscript(bands, sampleType):\n",
+    "    _input = str([band for band in bands]).replace(\"'\", '\"')\n",
```
Comment (zigaLuksic) [inline GEM-data/gem-collections.ipynb:62]: use `input_bands` or similar, the `_` prefix has a strong connotation in python, i think it's better if the names don't match 100% than to use it

**Review by zigaLuksic — COMMENTED** [review]
I cant see the diff on the last commit so ignore anything you already fixed.
After this is resolved I'll take a look at the entire notebook again :)

File: GEM-data/gem-collections.ipynb [hunk 2]
```diff
@@ -164,7 +167,38 @@
...
+    "                    np.clip(eopatch.data[\"bands\"][idx][:, :, [2, 1, 0]].squeeze() / 10000 * 3, 0, 1)\n",
+    "                )\n",
+    "            else:\n",
+    "                ax[idx // ncols][idx % ncols].imshow(eopatch.data[\"bands\"][idx][...].squeeze())\n",
+    "            ax[idx // ncols][idx % ncols].set_title(str(eopatch.timestamp[idx])[:10], size=12)\n",
+    "\n",
+    "        ax[idx // ncols][idx % ncols].get_xaxis().set_visible(False)\n",
+    "        ax[idx // ncols][idx % ncols].get_yaxis().set_visible(False)"
+   ]
+  },
```
[the remark this answers is not in the document]
Comment (jgersak) [reply GEM-data/gem-collections.ipynb]: done

File: GEM-data/gem-collections.ipynb [hunk 3]
```diff
@@ -51,6 +51,36 @@
...
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "e390008c-a274-46da-8521-b8f1e8c9e78a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def templat_evalscript(bands, sampleType):\n",
```
[the remark this answers is not in the document]
Comment (jgersak) [reply GEM-data/gem-collections.ipynb:61]: done

File: GEM-data/gem-collections.ipynb [hunk 4]
```diff
@@ -51,6 +51,36 @@
...
+    "    _input = str([band for band in bands]).replace(\"'\", '\"')\n",
+    "    evaluatePixel = str([(\"sample.\" + band) for band in bands]).replace(\"'\", \"\")\n",
```
[the remark this answers is not in the document]
Comment (jgersak) [reply GEM-data/gem-collections.ipynb:63]: done

File: GEM-data/gem-collections.ipynb (hunk 1 above)
Comment (jgersak) [reply GEM-data/gem-collections.ipynb:62]: done

**Pull request merged**